nossas/bonde
# Fix select widget to settings

## How to test

1. Go to 'Edit mobilization' page
2. With 2 widgets added of same kind in your mobilization, go to widget settings page
3. Should open settings page with data of selected widget
